src/rust/dbv2/migrations/2024-01-09-160715_price_to_nominal/up.sql

```diff
@@ -0,0 +1,14 @@
+-- Your SQL goes here
+CREATE FUNCTION get_price_divisor_for_market(numeric) RETURNS NUMERIC IMMUTABLE AS $$
+    SELECT COALESCE ((
+        SELECT 1 / ((tick_size / POW(10,quote.decimals)) / (lot_size / POW(10,base.decimals)))
+        FROM market_registration_events
+        INNER JOIN aggregator.coins base
+        ON base_account_address = address AND base_module_name = module AND base_struct_name = struct
+        INNER JOIN aggregator.coins quote
+        ON quote_account_address = address AND quote_module_name = module AND quote_struct_name = struct
+        WHERE market_id = $1
+        AND base_name_generic IS NULL
+    ),1);
+$$ LANGUAGE sql;
```

Comment:
How about?

- [ ] Rename `get_price_divisor_for_market` to `integer_price_to_nominal_price(integer_price, market_id)` (model off `decimals.py`, assume `base_decimals` is 1 when generic asset)
- [ ] Add `nominal_price_to_integer_price(nominal_price, market_id)` (model off `decimals.py`, assume `base_decimals` is 1 when generic asset)
- [ ] Add `size_to_base_indivisible_subunits(size, market_id)` (multiply by lot size)
- [ ] Add `size_and_price_to_quote_indivisible_subunits(size, integer_price, market_id)` (size times price is volume in ticks, then multiply by tick size to get quote indivisible subunits)